exporters/metric/cortex/cortex.go

```diff
@@ -254,7 +255,7 @@ func convertFromMinMaxSumCount(record metric.Record, minMaxSumCount aggregation.
 	}
 	countSample := prompb.Sample{
 		Value:     float64(count),
-		Timestamp: record.EndTime().Unix(), // Convert time to Unix (int64)
+		Timestamp: record.EndTime().UnixNano() / int64(time.Millisecond),
```

Comment:
Why are we changing nanoseconds to milliseconds, is this what the backend expects?

Reply:
Yeah, that was my mistake in not reading the docs carefully enough...